Heat-Bundle/pom.xml

```diff
@@ -2,34 +2,28 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>hcom.funcTest</groupId>
+        <groupId>com.hotels</groupId>
         <artifactId>Heat-Core</artifactId>
```

Comment:
All our other projects have followed the convention of the module names being all lower case, so these should become "heat-core", "heat-bundle" etc. I think this is easier for end users as they don't have to think about case and whether the names are case-sensitive or not.

Seeing as we're making so many changes we might as well go full on here ;)